app/registries/foreman/plugin/importer_registry.rb

```diff
@@ -0,0 +1,26 @@
+module Foreman
+  class Plugin
+    class ImporterRegistry
+      attr_accessor :importers 
+
+      def initialize
+        @importers = {}
+        @importers.default = 'PuppetFactImporter'
```

Comment:
Can we move the default setting to initializer? As a long-term goal, I want to move puppet to a plugin and then this line would not be possible completely.